modules/distr-tracing-accessing-jaeger-console.adoc

```diff
@@ -0,0 +1,60 @@
+////
+Module included in the following assemblies:
+
+////
+
+[id="distr-tracing-accessing-jaeger-console_{context}"]
+= Accessing the Jaeger console
+
+To access the Jaeger console you must have either {ProductName} or {DTProductName} installed, and {JaegerName} installed and configured.
```

Comment:
I think something is not going well here. [In the preview](https://deploy-preview-40980--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_install/distr-tracing-deploying-jaeger.html) I can see this:
```
To access the Jaeger console you must have either {ProductName} or Red Hat OpenShift distributed tracing installed, and Red Hat OpenShift distributed tracing platform installed and configured.
```
` {ProductName}` is not well interpolated

Reply:
I was going to suggest viewing the netlify preview instead of the GitHub preview, but it looks like there is at least one problem with a variable.  (https://deploy-preview-40980--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_install/distr-tracing-deploying-jaeger.html#distr-tracing-accessing-jaeger-console_deploying-distr-tracing-platform).  Thanks for catching that.  I made two passes through the output searching for broken variables and still missed that one!

Reply:
This is what is rendered: 

> To access the Jaeger console you must have either Red Hat OpenShift Service Mesh or Red Hat OpenShift distributed tracing installed, and Red Hat OpenShift distributed tracing platform installed and configured.

which does not seem to be correct. The Jaeger UI is available only if  `Red Hat OpenShift distributed tracing platform` is installed and an instance is created.

Reply:
Changed "is installed and an instance is created."  to "is installed, configured, and deployed."